b_lambda_layer_common/source/python/util/logging.py

```diff
@@ -0,0 +1,33 @@
+import logging
+from typing import Optional
+
+
+class LoggingManager:
+    '''
```

Comment:
This is not how docstrings are used in python. Please do some research about python docstrings.